### ISSUE TITLE
Feature request: Disabled item state

### ISSUE THREAD

**Issue by ricsam** [vonovak/react-navigation-header-buttons#19]
It would be nice to have an optional bool prop disabled on the HeaderButtons.Item component to style it accordingly and disable the animation

**Comment by vonovak** [comment]
hi! thanks for the feature suggestion and for your related PR. It definitely is a valid request.

I have already taken a quick look at it and I'm not sure what the best approach would be here (my goal is to have the api as simple as possible).

Adding the disabled state means adding two props - `disabled` for the button, and also additional style for the disabled state. Now, I want to provide this style only once, not repeat it for all `HeaderButton.Item`s. With the current way of doing things, that would mean passing the style to the `HeaderButtons` component, but I would ideally like to avoid passing props to `HeaderButtons` if they are really just passed through to `HeaderButton.Item`s - my point is, why pass a prop to `HeaderButtons` if it is intended for `HeaderButton.Item`? Ideally I'd like to wrap `HeaderButton.Item` in some other component. I'll think about this, thanks again!

**Comment by vonovak** [comment]
this is now available in v2, [se how](https://github.com/vonovak/react-navigation-header-buttons/blob/master/example/navbar-buttons-demo/screens/UsageDisabled.js)

**Comment by ricsam** [comment]
@vonovak Great job! 🎉 I think you implemented it in a nice and extensible way! 🏅
